cypress/support/commands.js

```diff
@@ -93,7 +93,7 @@ Cypress.Commands.add("nextPageByNav", (p_navItemName) => {
 });
 
 // Go through index page and select participants tsv and json dictionary files, advance to categorization page
-Cypress.Commands.add("passIndex", () => {
+Cypress.Commands.add("passIndex", (p_dataTableFilepath, p_dataDictionaryFilepath) => {
```

Comment:
These `pass` commands now take variable input paths from the current testing context.

Reply:
That's correct. The other reason I'm mentioning these functions - and I'll continue this thought in your other general comment on UI-based testing - is that these pass functions may be rendered moot depending on how we decide to configure the UI e2e test(s).

More on that in your general comment on the PR.